_posts/2021-11-10-release-0.9.13.md

```diff
@@ -0,0 +1,194 @@
+
+```py
+texture = carla.TextureColor(width,height)
+for x in range(0,len(image[0])):
+    for y in range(0,len(image)):
+        color = image[y][x]
+        r = int(color[0])
+        g = int(color[1])
+        b = int(color[2])
+        a = int(color[3])
+        texture.set(x, height - y - 1, carla.Color(r,g,b,a))
+world.apply_color_texture_to_object('SM_Cartel_Add_5', carla.MaterialParameter.Diffuse, texture, 0)
+```
+
+![runtime_texture_updating](/img/posts/2021-11-07/runtime_texture.gif){:class="img-fluid rounded mx-auto d-block"}
+
+## Pedestrian ground truth skeletons
+
+The CARLA API now provides functionality to retrieve the ground truth bone positions of pedestrians in the simulation. This opens up a wealth of possibilities for training and evaluation of human pose estimation models for use in autonomous driving. A CARLA vehicle can now record the bone positions of nearby pedestrians to compliment it's sensor data with access to the ground truth pose for comparison with model-estimated poses. This new functionality also allows easy augmentation of pedestrian behavior with custom guestures blended into the default animations like waving or pointing. 
+
+The following animation shows a custom animation handled by the Python API retrieving the bones then blending between the neutral pose and a custom walking animation handled through the API:
+
+![pedestrian_animation](/img/posts/2021-11-07/pedestrian.gif){:class="img-fluid rounded mx-auto d-block"}
+
+The following code gets a pedestrian's bones controlling the arms and adjusts the arms' pose:
+
+```py
+# get the bones
+bones = actor.get_bones()
+
+# modify some bones
+new_pose = []
+for bone in bones.bone_transforms:
+    if bone.name == "crl_foreArm__L":
+        bone.relative.rotation.pitch -= 90
+        new_pose.append((bone.name, bone.relative))
+    elif bone.name == "crl_foreArm__R":
+        bone.relative.rotation.pitch -= 90
+        new_pose.append((bone.name, bone.relative))
+
+# set the new pose
+control = carla.WalkerBoneControlIn()
+control.bone_transforms = new_pose
+actor.set_bones(control)
+
+# blend the pose
+actor.blend_pose(0.5)
+```
+
+## New CARLA assets
+
+### New pedestrians
+
+4 new pedestrians have been added to the CARLA assets. 2 adults (each with 3 variants) and 2 children. 
+
+![neew_pedestrians](/img/posts/2021-11-07/new_pedestrians.jpg){:class="img-fluid rounded mx-auto d-block"}
+
+### New volkswagen van
+
+The CARLA garage has built an exciting new vehicle, the __Volkswagen T2 2021__. Add some charm to your city traffic with this beautifully retro styled vehicle. 
```

Comment:
I don't really like this, as the Volkswagen T2 is already in CARLA. This is "just" another blueprint of the same model, but with a much higher visual fidelity.

Reply:
I'll add that it's a higher fidelity vehicle, nobody actually mentioned before that it wasn't a completely new vehicle